postgres-persistence/src/test/java/com/netflix/conductor/postgres/dao/PostgresPollDataDAOCacheTest.java

```diff
@@ -94,25 +94,11 @@ private List<Map<String, Object>> queryDb(String query) throws SQLException {
 
     private void waitForCacheFlush() throws InterruptedException {
         long startTime = System.currentTimeMillis();
-        long lastDiff =
-                System.currentTimeMillis() - ((PostgresPollDataDAO) pollDataDAO).getLastFlushTime();
+        long lastFlushTime = ((PostgresPollDataDAO) pollDataDAO).getLastFlushTime();
 
-        if (lastDiff == 0) {
-            return;
-        }
-
-        while (true) {
-            long currentDiff =
```

Comment:
This whole logic could be simplified a fair bit, so I went ahead and did that while I was trying to figure out what was going wrong.